graylog2-web-interface/src/components/sidecars/sidecars/SidecarStatus.jsx

```diff
@@ -97,6 +109,11 @@ const SidecarStatus = createReactClass({
     );
   },
 
+  _onShowVerbose(name, verbose) {
+    this.setState({collectorName: name, collectorVerbose: verbose});
```

Comment:
There's another ESLint rule for spaces around object curly braces: https://eslint.org/docs/rules/object-curly-spacing
```suggestion
    this.setState({ collectorName: name, collectorVerbose: verbose });
```